examples/demo_integrate_limits.py

```diff
@@ -64,17 +67,25 @@
     data /= ph0
     fl.image(data)
     data.ft("t2")
+    time_list.append(time.time())
     fl.next("fake data -- freq domain")
     fl.image(data)
     for method in ['Lorentzian','Gaussian']:
-        fl.basename = method
+        time_list.append(time.time())
+        fl.basename = method + " filter:"
+        logger.debug(strm("from main script, basename is",fl.basename,"and current",fl.current))
         freq_lim = integrate_limits(data['ph1',0]['ph2',1],
                 convolve_method=method,
                 fl=fl)
-        fl.next("fake data -- show freq limit selection, %s filter"%method)
+        logger.debug("about to run potentially problematic next command")
+        fl.next("fake data -- show freq limit selection")
         fl.plot(data['ph1',0]['ph2',1])
         axvline(x=freq_lim[0])
         axvline(x=freq_lim[-1])
         print("Determined frequency limits via",method,"filter of",freq_lim)
+    time_list.append(time.time())
+    print("figure list-----\n",fl) # there were extra/blank figures, and I added this to help me figure out which they were (update to pyspecdata needed)
```

Comment:
note this feature (I end up pulling it out b/c it was for debugging, but good to know about) --> I added this to pyspecdata today